platform-common-pdf/src/main/java/com/softicar/platform/common/pdf/PdfMerger.java

```diff
@@ -0,0 +1,51 @@
+package com.softicar.platform.common.pdf;
+
+import com.softicar.platform.common.io.stream.InputStreamCollection;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.function.Supplier;
+import org.apache.pdfbox.io.MemoryUsageSetting;
+import org.apache.pdfbox.multipdf.PDFMergerUtility;
+import org.apache.pdfbox.multipdf.PDFMergerUtility.DocumentMergeMode;
+
+/**
+ * Merges a {@link Collection} of PDFs into a single PDF.
```

Comment:
> Collection

Should we rather talk about a `List` to imply a well-defined order?

Reply:
- `List` not only implies order but also random access (indexed access) which is an unnecessary requirement
- changed to the generic term `sequence`

Reply:
True that. Never mind.